frontend/src/components/MeetingRoom/ArchivingButton/ArchivingButton.tsx

```diff
@@ -8,20 +8,27 @@ import { startArchiving, stopArchiving } from '../../../api/archiving';
 import useSessionContext from '../../../hooks/useSessionContext';
 import useIsSmallViewport from '../../../hooks/useIsSmallViewport';
 
+export type ArchivingButtonProps = {
+  handleCloseMenu?: () => void;
+};
+
 /**
  * ArchivingButton Component
  *
  * Displays a button and handles the archiving functionality. If a meeting is currently being recorded,
  * will confirm that a user wishes to stop the recording. If a meeting is not being recorded, prompts
  * the user before starting the archive.
+ * @param {ArchivingButtonProps} props - The props for the component.
+ *  @property {() => void} handleClickAway - (optional) click handler that closes the overflow menu in small view port devices.
  * @returns {ReactElement} - The ArchivingButton component.
  */
-const ArchivingButton = (): ReactElement => {
+const ArchivingButton = ({ handleCloseMenu }: ArchivingButtonProps): ReactElement => {
   const roomName = useRoomName();
   const { archiveId } = useSessionContext();
   const isRecording = !!archiveId;
   const [isModalOpen, setIsModalOpen] = useState<boolean>(false);
   const title = isRecording ? 'Stop recording' : 'Start recording';
+  const isSmallViewport = useIsSmallViewport();
```

Comment:
If yours is merged in after my overflow rework, please change this to use the `useShownButtons` hook 🙏